gno.land/cmd/gnoland/testdata/addpkg_outofgas.txtar

```diff
@@ -0,0 +1,57 @@
+# ensure users get proper out of gas errors when they add packages
+
+# start a new node
+gnoland start
+
+# add foo package
+gnokey maketx addpkg -pkgdir $WORK/foo -pkgpath gno.land/r/foo -gas-fee 1000000ugnot -gas-wanted 211483 -broadcast -chainid=tendermint_test test1
+
+
+# add bar package
+# out of gas at store.GetPackage() with gas 60000
+
+! gnokey maketx addpkg -pkgdir $WORK/bar -pkgpath gno.land/r/bar -gas-fee 1000000ugnot -gas-wanted 60000 -broadcast -chainid=tendermint_test test1
+
+# Out of gas error
+
+stderr '--= Error =--'
+stderr 'Data: out of gas error'
+stderr 'Msg Traces:'
+stderr '    0  /home/runner/work/gno/gno/tm2/pkg/crypto/keys/client/maketx.go:215 - deliver transaction failed: log:out of gas, gasWanted: 60000, gasUsed: 60343 location: ReadFlat'
```

Comment:
```suggestion
stderr '    0  /home/runner/work/gno/gno/tm2/pkg/crypto/keys/client/maketx.go:215 - deliver transaction failed: log:out of gas, gasWanted: 60000, gasUsed: \d+ location: ReadFlat'
```

Same for the other gas used. I'd rather we checked the values of gas in gas_test.go rather than in filetests.

Reply:
The chain consumes gas at multiple points outside the VM before reaching the VM. Even inside the VM, a transaction may run out of gas during preprocessing or runtime. To precisely replicate and verify that the bug is fixed in the preprocessing stage, we need to specify an exact gas amount in the integration test.

Reply:
This is checking against the error message. If we change how gas calculated, even slightly, this is yet another file where we need to go update the tests. It's an annoyance.

In this case, it's not necessary for the test; this is just checking the error message.

Reply:
Okay, we can use stderr with 'out of gas' instead to avoid failing on a specific gas value used in the error message. It also avoids failure cases on a local computer since the error message contains the path to the maketx.go file.

Reply:
Okay, we can use stderr with 'out of gas.*?in preprocess'  instead to avoid failing on a specific gas value used in the error message. It also avoids failure cases on a local computer, as the error message contains the path to the maketx.go file.